lib/double_entry/reporting.rb

```diff
@@ -78,8 +78,8 @@ class AggregateFunctionNotSupported < RuntimeError; end
     # @raise [Reporting::AggregateFunctionNotSupported] The provided function
     #   is not supported.
     #
-    def aggregate(function, account, code, range, options = {})
-      Aggregate.formatted_amount(function, account, code, range, options)
+    def aggregate(function, account, code, range, filter: nil, partner_account: nil)
```

Comment:
Ah did we?  I forgot about that conversation!

I suppose either way people will have to step up to this change in some way (such as running migrations)...so we might as well do it now.